lib/chartjs.dart

```diff
@@ -863,6 +863,7 @@ abstract class ChartTooltipOptions {
       dynamic /*String|CanvasGradient|CanvasPattern|List<String>*/ v);
   external num get borderWidth;
   external set borderWidth(num v);
+  external factory ChartTooltipOptions();
```

Comment:
I know it's a bit tedious, but could you also add all the fields in the constructor as optional named arguments? thanks!

Reply:
Sorry yes, added it